docker_cli/_image.py

```diff
@@ -33,7 +32,7 @@ def __init__(self, name_or_id: str):
         Parameters
         ----------
         name_or_id : str
-            A name or ID by which to find this image using docker inspect.
+            A name or ID by which to find this image using Docker inspect.
```

Comment:
Here we're referring to the `docker inspect` command (rather than the "Docker" project itself) so we should use a lowercase "d" in this particular case.

```suggestion
            A name or ID by which to find this image using `docker inspect`.
```